src/Entities/User.java

```diff
@@ -0,0 +1,52 @@
+package Entities;
+
+import Entities.Address;
+
+public abstract class User {
+    protected int id;
```

Comment:
folosesti vreodata direct aceste campuri in clasele copil? daca nu, nu e nevoie sa fie protected, pot fi private, chit ca este clasa extinsa

Reply:
Da, le folosesc cand dau override la toString.